testsuite/MDAnalysisTests/analysis/test_base.py

```diff
@@ -95,6 +95,12 @@ def test_verbose(u):
     assert a._verbose
 
 
+def test_verbose_run(u):
+    a = FrameAnalysis(u.trajectory, verbose=False).run(verbose = True)
+    assert not a._verbose
```

Comment:
`assert a._verbose is False`

Reply:
`not a._verbose` is correct. There is no reason to compare the identity with the `False` singleton; in theory, everything that is considered false by python would be a valid value here.

Reply:
Correct. Thanks for the feedback!